rest-server/src/lib/log/index.js

```diff
@@ -0,0 +1,9 @@
+import * as chalk from 'chalk';
+
+export const success = (...log) => {
+  if (process.env.DEBUG === 'TRUE') console.log(chalk.default.white.bgGreen.bold(...log));
```

Comment:
What happens when debug isn't set or does not evaluate to `TRUE`?

Reply:
This is a mistake on my part. I should have accounted for that. While the server runs, the server console logs stop displaying altogether if the env property does not evaluate to TRUE. I will fix this.